modules/microshift-data-cleaning-certs.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-certs_{context}"]
+= Cleaning the certificates data
+
+Run the following command to cleanup {microshift-short} certificates and to reset {microshift-short} custom certificates so they recreate when the {microshift-short} service restarts. This command performs the following cleanup actions:
+
+* Stop and disable {microshift-short} services
+* Delete all {microshift-short} pods
+* Delete all {microshift-short} certificates
+
+.Procedure
+
+. Run the following command to cleanup {microshift-short} certificates.
++
+[source,terminal]
+----
+$ sudo microshift-cleanup-data --cert
+----
++
+.Example output
+[source,terminal]
+----
+Stopping MicroShift services
+Removing MicroShift pods
+Removing MicroShift certificates
+MicroShift service was stopped
+Cleanup succeeded
+----
+
+. Run the following command to restart the {microshift-short} service.
```

Comment:
Please, copy the full explanation from the doc.

Reply:
done. note is added